pkg/labels/labels.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package labels
```

Comment:
I think `labels` is a more readable package name, why alias it to `labelutil` at the call sites?

Reply:
@youngnick thanks for the review. Commit `5876131` removes the `labelutil` pkg alias for all imports.